api/tools/src/main/java/org/pmiops/workbench/tools/cdrConfig/AccessTierExtractor.java

```diff
@@ -0,0 +1,13 @@
+package org.pmiops.workbench.tools.cdrConfig;
+
+import java.util.List;
+import org.pmiops.workbench.db.model.DbAccessTier;
+
+// expected format:
+// { "accessTiers": [ {DbAccessTier}, ...], "cdrVersions": [ {DbCdrVersion}, ...] }
+
+// however we need to parse them separately because we use an adapter to resolve the accessTier
```

Comment:
Maybe there's a better way to do this besides parsing 3 times using 3 different extractors...

Reply:
Another way you could consider doing this would be to define a POJO for the entire config model, rather than directly referencing DB  models. Gson parsing straight into the DB model was convenient, but we were already hitting issues with it (e.g. ideally we would allow you to specify enum names, rather than using ordinals in the config).

This would create some duplication, but conversion should be straightforward, and could potentially use mapstruct (this would also automatically catch divergence). In the POJO you would just define the field as a long to make parsing simple (or use the shortname, if that's more legible).

Reply:
Yeah I think I will give that a try.  I started down that road on a previous iteration.

Reply:
It works, and is also a much cleaner design.  Thanks!

One issue is that we don't currently have MapStruct working in the tools dir, so I moved that part back into the main app dir.  If we can't resolve this quickly I'd prefer to keep this as a TODO.  I don't really know gradle so my initial attempt to add this was unsuccessful.